ui/src/types/pdfjs-viewer.ts

```diff
@@ -38,6 +38,7 @@ export interface DocumentLoadedEvent {
 export interface PDFViewer {
   scrollPageIntoView: (params: ScrollPageIntoViewParameters) => void;
   container: HTMLDivElement;
+  viewer: HTMLDivElement;
```

Comment:
@andrewhead not sure why this was removed, but I need it to add the padding. Hope it's not an issue to add back.